README.md

```diff
@@ -207,16 +207,16 @@ You can also add `+yyyymmdd` after `torch_xla-nightly` to get the nightly wheel
 of a specified date. To get the companion pytorch and torchvision nightly wheel,
 replace the `torch_xla` with `torch` or `torchvision` on above wheel links.
 
-#### Installing libtpu (before PyTorch/XLA 2.0)
+#### Installing libtpu
```

Comment:
Can you just delete this section? There are more up to date instructions to install libtpu up top.

Reply:
Done.